pages.es/common/git-revert.md

```diff
@@ -13,7 +13,7 @@
 
 - Revierte múltiples commits:
 
-`git revert {{master~5..master~2}}`
+`git revert {{branch_name~5..branch_name~2}}`
```

Comment:
Same as [above](https://github.com/tldr-pages/tldr/pull/4533/files#r500594375) — these should be translated.

Reply:
```suggestion
`git revert {{rama~5..rama~2}}`
```